.github/workflows/downsize-images.yml

```
@@ -0,0 +1,70 @@
+name: downsize-images
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  DownsizeImages:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v2
+    - name: Install imageMagick
+      run: |
+          echo "Installing ImageMagick"
+          wget https://imagemagick.org/download/binaries/magick
+          chmod +x magick         
+          sudo mv magick /usr/local/bin
+          which magick
+
+    - name: Run image downsizing script
+      run: /bin/bash scripts/downscale_images.sh
+
+    - name: Checkout New Branch
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        BRANCH_AGAINST: "master"
+      run: |
+        printf "GitHub Actor: ${GITHUB_ACTOR}\n"
+        export BRANCH_FROM="downsize-images/$(date '+%Y-%m-%d/%H-%M-%S')"
+        git remote set-url origin "https://x-access-token:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git"
+
+        # Test if branch exists in remote
+        BRANCH_EXISTS=$(git ls-remote --heads origin ${BRANCH_FROM})
+        if [[ -z ${BRANCH_EXISTS} ]]; then
+            printf "Branch does not exist in remote.\n"
+        else
+            printf "Branch already exists in remote.\n"
+            exit 1
+        fi
+        git branch
+        git checkout -b "${BRANCH_FROM}" || git checkout "${BRANCH_FROM}"
+        git branch
+
+        git config --global user.name "github-actions"
+        git config --global user.email "github-actions@users.noreply.github.com"
+
+        git status
+        git add assets/*
+
+        if git diff-index --quiet HEAD --; then
+           printf "No changes\n"
+           export OPEN_PULL_REQUEST=0
+        else
+           export OPEN_PULL_REQUEST=1
+           printf "Changes\n"
+           git commit -m "Automated deployment to downsize images $(date '+%Y-%m-%d')"
+           git push origin "${BRANCH_FROM}"
+        fi
+        echo "::set-env name=OPEN_PULL_REQUEST::${OPEN_PULL_REQUEST}"
+        echo "::set-env name=PULL_REQUEST_FROM_BRANCH::${BRANCH_FROM}"
+        echo "::set-env name=PULL_REQUEST_BODY::Downsizing images automated pull request."
```

```suggestion
        echo "::set-env name=PULL_REQUEST_TITLE::[downscale-images] ${BRANCH_FROM}"
        echo "::set-env name=PULL_REQUEST_BODY::Downsizing images automated pull request."
```
I'd change the title of the PR to indicate that it's generated by a bot, but otherwise, please merge =)

---

I think we'd want to test this before just merging, in case there is an issue with the characters `[` and `]`. I'm happy to commit this, and then have the "test" be an actual run. Please commit that and merge if you want to take that approach, otherwise just merge (this was my PR so it's better if you merge it into your branch!)